changelog.yaml

```diff
@@ -14,9 +14,18 @@
 
 #      jiraIssueNumber: <Internal Jira issue number>
 
-- version: "NEXT"
-  date: TBD
-  
+- version: "0.6.0"
+  date: 2020-10-07
+  changes:
+  - type: enhancement
+    impact: minor
+    title: Make Jenkinsfile Runner container image configurable in PipelineRun custom resource objects.
+
+    description: >-
+      Jenkins filerunner image is configurable in Steward-PipelineRun manifest at 'spec.image' now.
+    pullRequestNumber: 162
+    jiraIssueNumber: 983
+

```

Comment:
We should consider keeping `warning` and `upgradeNotes` in the changelog.yaml. If it is not required we should assign an empty string else the required action(s).

Reply:
Done

Reply:
Done